daemon/daemon_unix.go

```diff
@@ -1151,3 +1154,20 @@ func setupOOMScoreAdj(score int) error {
 	f.Close()
 	return err
 }
+
+func (daemon *Daemon) setupSeccompProfile() error {
+	b, err := ioutil.ReadFile(seccompProfilePath)
+	if err != nil {
+		// TODO(runcom): generate a default seccomp profile if system wide isn't present?
```

Comment:
Currently I don't think we provide ways to override a lot of the file locations for things. I would think that if we provided a way to override the `/etc/docker` location in future, this would then look in that directory, but I don't think it needs to be explicitly overrideable other than that.